app/src/sharedTest/java/org/oppia/android/app/databinding/AppCompatCheckBoxBindingAdaptersTest.kt

```diff
@@ -127,7 +127,7 @@ class AppCompatCheckBoxBindingAdaptersTest {
     activityRule.scenario.onActivity {
       val appCompatCheckBox: AppCompatCheckBox = getAppCompatCheckBox(it)
       setButtonTint(appCompatCheckBox, colorRgb)
-      assertThat((appCompatCheckBox.buttonTintList)?.defaultColor).isEqualTo((colorRgb))
+      assertThat((appCompatCheckBox.supportButtonTintList)?.defaultColor).isEqualTo((colorRgb))
```

Comment:
What ended up being the issue @Akshatkamboj14?

Reply:
actually @BenHenning, I was calling setSupportButtonTintList in the binding adapter, and in the test I was calling getButtonTintList so I think that was the issue, when I changed this (buttonTintList to supportButtonTintList) its calling the get method of SupportButtonTintList and the test is passing.

Reply:
and moreover ben, there's is some error in binding adapter and this error is the reason of the failing Bazel test, can you help me in this?

Reply:
Left one follow-up comment--PTAL.

Reply:
Okay, Ben.